Cargo.toml

```diff
@@ -43,3 +43,16 @@ features = ["vulkan"]
 simple_logger = "1.3"
 approx = "0.3"
 serde = { version = "1.0", features = ["derive"] }
+
+[[example]]
+name = "basic"
+path = "examples/basic.rs"
+
+[[example]]
+name = "batch"
+path = "examples/batch.rs"
+
+[[example]]
+name = "amethyst"
+path = "examples/amethyst/main.rs"
+required-features = ["amethyst"]
```

Comment:
If we are to explicitly add the examples to the manifest, could you also add `dim3` to the required features for each of them? I intend to make this feature non-default on release.

Reply:
sure.